server/src/fhir/definitions.ts

```diff
@@ -1,38 +1,64 @@
-import NodeCache from 'node-cache'
 import set from 'lodash.set'
 import axios from 'axios'
 
 import { StructureDefinition } from 'types'
 import { FHIR_API_URL } from '../constants'
-
-const cache = new NodeCache()
+import cache from 'cache'
 
 const metaPrefix = '$meta'
 
 // Gets a definition from the cache.
-export const getDefinition = (key: string): StructureDefinition | undefined =>
-  cache.get(key)
+export const getDefinition = async (
+  key: string,
+): Promise<StructureDefinition | undefined> => {
+  const { get } = cache()
+  const res = await get(key)
+  return res ? JSON.parse(res) : undefined
```

Comment:
Are you doing this because we can't put objects in the cache?

Reply:
yes!